src/main/java/controller/LottoGameController.java

```diff
@@ -1,38 +1,63 @@
 package controller;
 
-import domain.AnswerLotto;
-import domain.BonusNumber;
-import domain.AnswerLottoNumbers;
-import domain.Lottos;
-import util.RandomLottoNumberGenerator;
+import domain.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
 import static view.InputView.*;
 import static view.OutputView.*;
 
 public class LottoGameController {
+	private final ArrayList<Integer> LottoNumberCandidates = new ArrayList<>();

```

Comment:
```suggestion
	private final List<Integer> LottoNumberCandidates = new ArrayList<>();

```
구현클래스가 아닌 인터페이스를 사용하여 선언하면 어떤 장점이 있을까요?

Reply:
추후에 구현클래스가 변경되더라도 해당 클래스를 이용하는 곳에서는 코드의 변화가 일어나지 않습니다! 알고있었는데 놓쳤네요ㅠ 더 꼼꼼하게 확인하겠습니다!